xarray/convert.py

```diff
@@ -38,15 +39,28 @@ def from_cdms2(variable):
     """
     values = np.asarray(variable)
     name = variable.id
-    coords = [(v.id, np.asarray(v),
-               _filter_attrs(v.attributes, cdms2_ignored_attrs))
-              for v in variable.getAxisList()]
+    dims = variable.getAxisIds()
+    coords = {}
```

Comment:
Can you switch this to `OrderedDict` instead? We usually try to use those to keep the order of coordinates more predictable (though it's increasingly less relevant with ordered dicts by default on Python 3).